docs/SUPPORTED_BROWSERS.md

```diff
@@ -0,0 +1,31 @@
+# Supported Browsers
+
+## Policy
+
+uPortal supports **the latest general audience release** and **one prior release** of these major browsers:
+
++ Chrome (on Android, MacOS, Windows) (including support for TalkBack)
++ Firefox (on MacOS and on Windows)
```

Comment:
Could you add Linux to Firefox ? something like:
`Firefox (on MacOS, IOS, Android, Windows, Linux) (including support for ChromeVox)`
Sometimes Linux users get upset very very quickly if they feel they are being forgotten.